aries-site/src/data/structures/components.js

```diff
@@ -821,6 +821,37 @@ export const components = [
     relatedContent: ['Page Layouts', 'Content Layouts', 'Box', 'Main'],
     tags: ['page', 'page content'],
   },
+  {
+    name: 'PageHeader',
+    available: true,
+    category: 'Layouts',
+    description: `A component providing users a consistent, predictable layout for locating a page's title, contextual navigation, and actions across all HPE applications.`,
+    preview: {
+      image: {
+        src: {
+          light: '/componentImages/component-preview-pageheader.svg',
+          dark: '/componentImages/component-preview-pageheader-invert.svg',
+        },
+        alt: 'HPE PageHeader Preview',
+        fit: 'contain',
+      },
+    },
+    seoDescription: `A required page component that provides users context of 
```

Comment:
should this be `A component providing users a consistent, predictable layout for locating a page's title, contextual navigation, and actions across all HPE applications.`